### PR TITLE
fix: dashboard multitenancy updates

### PR DESCRIPTION
**Pull request by sattvikc**
## Summary of change

(A few sentences about this PR)

## Related issues

-   https://github.com/supertokens/supertokens-node/pull/471

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [ ] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [ ] Had run `npm run build-pretty`
-   [ ] Had installed and ran the pre-commit hook
-   [ ] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If have added a new web framework, update the `add-ts-no-check.js` file to include that
-   [ ] If added a new recipe / api interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If added a new recipe, then make sure to expose it inside the recipe folder present in the root of this repo. We also need to expose its types.

## Remaining TODOs for this PR

-   [ ] Item1
-   [ ] Item2

### CODE REVIEW

File: lib/ts/recipe/dashboard/api/usersGet.ts [hunk 1]
```diff
@@ -47,7 +47,12 @@ export type Response = {
     }[];
 };
 
-export default async function usersGet(_: APIInterface, options: APIOptions, userContext: any): Promise<Response> {
```
Comment (rishabhpoddar) [inline lib/ts/recipe/dashboard/api/usersGet.ts:50]: this API's Response type should also contain tenantIds: string[]

File: lib/ts/recipe/dashboard/api/usersCountGet.ts [hunk 2]
```diff
@@ -21,7 +21,12 @@ export type Response = {
     count: number;
 };
 
-export default async function usersCountGet(_: APIInterface, __: APIOptions, ___: any): Promise<Response> {
+export default async function usersCountGet(
+    _: APIInterface,
+    ____: string,
+    __: APIOptions,
+    ___: any
+): Promise<Response> {
```
Comment (rishabhpoddar) [inline lib/ts/recipe/dashboard/api/usersCountGet.ts:29]: needs to change to be per tenant

File: lib/ts/recipe/dashboard/api/userdetails/userSessionsGet.ts [hunk 3]
```diff
@@ -30,7 +31,7 @@ export const userSessionsGet: APIFunction = async (
         });
     }
 
-    const response = await Session.getAllSessionHandlesForUser(userId, userContext);
+    const response = await Session.getAllSessionHandlesForUser(userId, undefined, userContext);
```
Comment (rishabhpoddar) [inline lib/ts/recipe/dashboard/api/userdetails/userSessionsGet.ts:34]: search for getAllSessionHandlesForUser in the SDK and make sure that we are passing 3 args everywhere

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/recipe/dashboard/api/usersGet.ts (hunk 1 above)
Comment (sattvikc) [reply lib/ts/recipe/dashboard/api/usersGet.ts:50]: Done

File: lib/ts/recipe/dashboard/api/userdetails/userSessionsGet.ts (hunk 3 above)
Comment (sattvikc) [reply lib/ts/recipe/dashboard/api/userdetails/userSessionsGet.ts:34]: Done, didn't find any other

**Pull request merged**